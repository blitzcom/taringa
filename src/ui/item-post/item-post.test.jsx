import renderer from 'react-test-renderer';

import ItemPost from './item-post';

describe('ui/item-post', () => {
  def('props', () => ({ title: 'foo', summary: { excerpt: 'bar' }, id: 'baz' }));
  def('subject', () => renderer.create(<ItemPost {...$props} />).toJSON());

  it('renders item post', () => {
    expect($subject).toMatchSnapshot();
  });
});
