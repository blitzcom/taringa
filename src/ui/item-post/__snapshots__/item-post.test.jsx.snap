// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/item-post renders item post 1`] = `
<article>
  <h2>
    foo
  </h2>
  <p>
    bar
  </p>
</article>
`;
